freezeyt/freezer.py

```diff
@@ -148,6 +148,7 @@ def handle_urls(self):
                 'PATH_INFO': encode_wsgi_path(path_info),
                 'SCRIPT_NAME': encode_wsgi_path(self.prefix.path),
                 'SERVER_PROTOCOL': 'HTTP/1.1',
+                'SERVER_SOFTWARE': 'freezeyt/0.0',
```

Comment:
I'd like to ask about the version number, here it is `0.0`, but in [setup.py](https://github.com/encukou/freezeyt/blob/master/setup.py#L5) it is version `0.1`.